pkg/providers/tinkerbell/reconciler/reconciler.go

```diff
@@ -31,17 +39,20 @@ type IPValidator interface {
 	ValidateControlPlaneIP(ctx context.Context, log logr.Logger, spec *c.Spec) (controller.Result, error)
 }
 
+type clusterChange string
```

Comment:
What's the motivation to unexport this?

Reply:
to use it as enum https://github.com/aws/eks-anywhere/pull/4971#discussion_r1113710109